sources/Application/Model/Project.h

```diff
@@ -17,7 +17,7 @@
 
 #define PROJECT_NUMBER "1.0"
 #define PROJECT_RELEASE "r"
-#define BUILD_COUNT "003"
+#define BUILD_COUNT "003a"
```

Comment:
we want to cut a new version with this? If we are working on master, I feel it's better to make feature/fix commits separate from a specific version, and use a specific new version commit when we decide to make one.

Do we want to make a new version out of this? in this case I guess it's ok, but I wouldn't keep adding sub versions, make it 004?

Reply:
Oh sorry thats my oversight, it was only temp when I made the quick hardcoded hack so I could tell which one I was testing. I'll take it out, good catch!